### ISSUE TITLE
ff CLI example errors with "lookup input does not exist in table"

### ISSUE THREAD

**Issue by gubsheep** [zkonduit/ezkl#47]
I attempted to run 

```
cargo run --release --bin ezkl -- mock -D ./examples/onnx_models/ff_input.json -M ./examples/onnx_models/ff.onnx
```

in CLI, following instructions in the README. Got this error:

```
error: lookup input does not exist in table
  (L0, L1) ∉ (F0, F1)

  Lookup 'lk' inputs:
    L0 = x1 * x0 + (1 - x1) * 0
    ^
    | Cell layout in region 'Elementwise':
    |   | Offset | A1 | F2 |
    |   +--------+----+----+
    |   |    0   | x0 | x1 | <--{ Lookup 'lk' inputs queried here
    |
    | Assigned cell values:
    |   x0 = 0x29d8
    |   x1 = 1

    L1 = x1 * x0 + (1 - x1) * 0
    ^
    | Cell layout in region 'Elementwise':
    |   | Offset | A1 | F2 |
    |   +--------+----+----+
    |   |    0   |    | x1 | <--{ Lookup 'lk' inputs queried here
    |   |    1   | x0 |    |
    |
    | Assigned cell values:
    |   x0 = 0x54
    |   x1 = 1

error: lookup input does not exist in table
  (L0, L1) ∉ (F0, F1)

  Lookup 'lk' inputs:
    L0 = x1 * x0 + (1 - x1) * 0
    ^
    | Cell layout in region 'Elementwise':
    |   | Offset | A2 | F2 |
    |   +--------+----+----+
    |   |    0   | x0 | x1 | <--{ Lookup 'lk' inputs queried here
    |
    | Assigned cell values:
    |   x0 = 0x40000000000000000000000000000000224698fc094cf91b992d30ecffffa742
    |   x1 = 1

    L1 = x1 * x0 + (1 - x1) * 0
    ^
    | Cell layout in region 'Elementwise':
    |   | Offset | A2 | F2 |
    |   +--------+----+----+
    |   |    0   |    | x1 | <--{ Lookup 'lk' inputs queried here
    |   |    1   | x0 |    |
    |
    | Assigned cell values:
    |   x0 = 0
    |   x1 = 1

thread 'main' panicked at 'circuit was not satisfied', /Users/bgu/.cargo/git/checkouts/halo2-afe2d0b0be6b3c3a/0cccba0/halo2_proofs/src/dev.rs:1292:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by alexander-camuto** [comment]
Hey @gubsheep thanks for flagging this. Can you let me know if the example works locally with the following parameters added to the command ? : 

`cargo run --release --bin ezkl -- --bits 16 -K 17 mock -D ./examples/onnx_models/ff_input.json -M ./examples/onnx_models/ff.onnx`

**Comment by alexander-camuto** [comment]
resolved by #49
